experimental/LieAlgebras/src/LieAlgebra.jl

```diff
@@ -437,6 +437,45 @@ Return `true` if `L` is solvable, i.e. the derived series of `L` terminates in $
   return dim(derived_series(L)[end]) == 0
 end
 
+###############################################################################
+#
+#   Root system getters
+#
+###############################################################################
+
+@doc raw"""
+    has_root_system(L::LieAlgebra) -> Bool
+
+Return whether a root system for `L` is known.
+
+This function should be implemented by subtypes that support root systems.
```

Comment:
Another off-topic comment, but: perhaps we should agree to whom docstrings of exported (documented) functions are addressed. I'd say it should be users. Of course sometimes there is also a point in adding some remarks for *implementors*. But I think those then should be marked explicitly as such, e.g. via a callout (so the `!!!` syntax). That said, in this case I think one could just rephrase it slightly:
```suggestion
This is only implemented for Lie algebras that support root systems.
```
or so?

Reply:
Thanks for the remark. I collected the information about which functions need to be implemented by subtypes that support root systems to the general list about subtype-implemented functions to the top. Furthermore, the docstrings are adapted to reflect what users need to know. (Note that I don't have to mention subtypes with no root system support there, as in these cases `has_root_system` should always return `false`, i.e. the `NotImplemented()` should never trigger.). Changes in 7ea927d5e34d33d0677ec409a5c5e01b1c1e9885